src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -216,7 +217,8 @@ public void testPostUpdateUserInfo() throws Exception {
 
   /** Helper method to add a test user to the local datastore */
   private void addTestUserEntityToDatastore(DatastoreService datastore) {
-    Entity userEntity = new Entity(TEST_USER_ID);
+    //Entity userEntity = new Entity(TEST_USER_ID);
+    Entity userEntity = new Entity("User");
```

Comment:
I would assume you'd want `new Entity(USER_ENTITY);`?

Reply:
Good catch, I'll fix that!